src/colorizer/ColorizeCanvas3D.ts

```diff
@@ -404,9 +404,9 @@ export class ColorizeCanvas3D implements IInnerRenderCanvas {
       const settings = channelSettings[backdropIdx];
       const channelIndex = backdropIndexToChannelIndex[backdropIdx];
       const colorArr: [number, number, number] = [
-        settings.color.r * 255,
-        settings.color.g * 255,
-        settings.color.b * 255,
+        settings.color.r * 255 * settings.opacity,
+        settings.color.g * 255 * settings.opacity,
+        settings.color.b * 255 * settings.opacity,
```

Comment:
I forgot to apply opacity settings to the color! I will note that this doesn't "look" correct, since Vol-E currently doesn't account for opacity. Would be great to update Vol-E's rendering in the future.